app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -134,5 +141,26 @@ data class OverdueAppointment(
         facilityUuid: UUID,
         scheduledBefore: LocalDate
     ): Observable<List<OverdueAppointment>>
+
+    @Query("""
+      $OVERDUE_APPOINTMENTS_QUERY
+      WHERE
+        P.uuid IN (
+          SELECT uuid FROM PatientFts WHERE fullName MATCH "*"||:query||"*"
+          UNION
+          SELECT P.uuid WHERE P.addressUuid = (
+            SELECT uuid FROM PatientAddressFts WHERE colonyOrVillage MATCH "*"||:query||"*"
```

Comment:
Instead of creating another sublist with this nested subquery. Could we have used the `OR` condition instead? For example


```sql
WHERE
  (
     P.uuid IN (SELECT uuid FROM PatientFts WHERE fullName MATCH "*"||:query||"*") OR
     P.addressUuid IN (SELECT uuid FROM PatientAddressFts WHERE colonyOrVillage MATCH "*"||:query||"*")
  )
```

WDYT?